beeline-examples/beeline-examples.cabal

```diff
@@ -37,9 +37,19 @@ library
     , beeline-routing >=0.2.1 && <0.3
     , shrubbery >=0.1 && <0.3
   default-language: Haskell2010
-  if flag(strict)
-    ghc-options: -Weverything -Werror -Wno-missing-local-signatures -Wno-monomorphism-restriction -Wno-missing-kind-signatures -Wno-prepositive-qualified-module -Wno-implicit-prelude -Wno-safe -Wno-unsafe -Wno-missing-safe-haskell-mode -Wno-missing-deriving-strategies -Wno-all-missed-specialisations -Wno-missed-specialisations -Wno-unticked-promoted-constructors
+  if flag(ci)
+    ghc-options: -O2 -Wall -Werror -Wcompat -Widentities -Wmissing-local-signatures -Wmissing-export-lists -Wpartial-fields -Wmissed-specialisations -Wunused-packages -Wno-implicit-prelude -Wno-safe -Wno-unsafe
+    if impl (ghc >= 9.0)
+      ghc-options: -Winvalid-haddock
+    if impl (ghc <= 9.3)
+      ghc-options: -Wno-unticked-promoted-constructors
+    if impl (ghc >= 9.0)
+      ghc-options: -Winvalid-haddock
```

Comment:
Looks like this block occurs twice, both on lines 42-43 and here on 46-47

Reply:
This is the exactly the thing mentioned below.